bootstrap.yml

```diff
@@ -65,11 +65,39 @@ sources:
     tag: 'binutils-2_32'
     version: '2.32'
     tools_required:
+      # TODO Dennis: autoconf-2.71 is too new, I'll grab this during the toolchain upgrade.
       - host-autoconf-v2.69
-      - host-automake-v1.15
+      - host-automake-v1.16
+      - host-libtool
     regenerate:
-      - args: ['autoreconf']
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/opcodes'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/opcodes'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/bfd'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/bfd'
+      - args: ['autoreconf', '-fvi', '-I../config']
         workdir: '@THIS_SOURCE_DIR@/ld'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/ld'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/gas'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/gas'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/binutils'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/binutils'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/gprof'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/gprof'
```

Comment:
Why is this done?

Reply:
According to mint, this is the better way of doing it. I tried not doing it in all directories but that will give libtool version mismatches. See also [this](https://github.com/managarm/bootstrap-managarm/pull/210#discussion_r986438187) comment thread.

Reply:
this is another reason to invoke the right version of the right tool, and that alone, for regenerating gcc files (remember, we aren't dealing with a normal autotools build system, but a broken one)

Reply:
Yeah not gonna carry libtool 2.2.7, gcc can suck it.

Reply:
didn't say that you should, neither autoconf and automake install libtool, this is only a problem if you don't use those (on top of the other problems that using wrong versions will cause)